_posts/2020-05-04-introducing-open-sgid.md

```diff
@@ -44,7 +44,7 @@ A nearly month-long beta testing period was helpful to identify several issues a
 - [Null value sorting behavior](https://github.com/agrc/open-sgid/issues/6)
 - [Required  selection of unique ID field in ArcMap](https://github.com/agrc/open-sgid/issues/9)
 
-If you come across a problem with or have a question about the Open SGID, visit the [issues page](https://github.com/agrc/open-sgid/issues). This repository not only acts as an important resource for Open SGID users to search for information about known issues, but also allows users to submit new issues they encounter, so AGRC staff and the general user population can collectively offer feedback and solutions. If you have not yet found an answer or solution to your topic, you can use the green _New issue_ button to submit it to the community. Also be sure to check out the [Open SGID Policy and Terms of Use page]({% about/policy/open-sgid/index.md %}), which provides additional information. 
+If you come across a problem with or have a question about the Open SGID, visit the [issues page](https://github.com/agrc/open-sgid/issues). This repository not only acts as an important resource for Open SGID users to search for information about known issues, but also allows users to submit new issues they encounter, so AGRC staff and the general user population can collectively offer feedback and solutions. If you have not yet found an answer or solution to your topic, you can use the green _New issue_ button to submit it to the community. Also be sure to check out the [Open SGID Policy and Terms of Use page]({% link about/policy/open-sgid/index.md %}), which provides additional information. 
 
 
 Although it may take some time for traditional Esri clients who are familiar with the SQL SDE database to adjust to the PostgresSQL database format, we view the Open SGID as an enhancement to AGRC’s data access points. It provides a direct SGID database connection that is no longer constrained to the Utah state government network, thereby living up to its name as a database that is truly open and accessible to all GIS platforms and users. We look forward to your feedback as both first-time and long-time SGID users utilize Utah’s most comprehensive and authoritative open-source geospatial database for your cartographic and analytical workflows.
```

Comment:
```suggestion
### A Truly Open Database

Although it may take some time for traditional Esri clients who are familiar with the SQL SDE database to adjust to the PostgresSQL database format, we view the Open SGID as an enhancement to AGRC’s data access points. It provides a direct SGID database connection that is no longer constrained to the Utah state government network, thereby living up to its name as a database that is truly open and accessible to all GIS platforms and users. We look forward to your feedback as both first-time and long-time SGID users utilize Utah’s most comprehensive and authoritative open-source geospatial database for your cartographic and analytical workflows.
```

Reply:
you can just make a commit to fix it, you don't have to use the suggestion since everyone already agreed on the change.